### PR TITLE
Sidemenu fixed for jwt

### PR DESCRIPTION
**Pull request by shishiro26**
## What kind of change does this PR introduce?
#18657 
Bug fix, feature, docs update, ...

## What is the current behavior?
problem siderbar

Please link any relevant issues here.

## What is the new behavior?
This pull request fixes the problem of the wrong sidebar 
fixed sidebar

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/components/interfaces/Database/Hooks/EditHookPanel.tsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ const EditHookPanel = ({ visible, selectedHook, onClose }: EditHookPanelProps) =
   const [isSubmitting, setIsSubmitting] = useState(false)
   const { mutate: createDatabaseTrigger } = useDatabaseTriggerCreateMutation({
     onSuccess: (res) => {
+      console.log('This is the response in new hook', res)
```
Comment (MildTomato) [inline studio/components/interfaces/Database/Hooks/EditHookPanel.tsx:56]: probably need to remove this

File: studio/components/interfaces/Database/Hooks/EditHookPanel.tsx [hunk 2]
```diff
@@ -71,11 +72,12 @@ const EditHookPanel = ({ visible, selectedHook, onClose }: EditHookPanelProps) =
   })
   const { mutate: updateDatabaseTrigger } = useDatabaseTriggerUpdateMutation({
     onSuccess: (res) => {
-      setIsSubmitting(false)
+      console.log( res)
```
Comment (MildTomato) [inline studio/components/interfaces/Database/Hooks/EditHookPanel.tsx:75]: probably need to remove this

File: studio/components/interfaces/Database/Hooks/EditHookPanel.tsx [hunk 3]
```diff
@@ -71,11 +72,12 @@ const EditHookPanel = ({ visible, selectedHook, onClose }: EditHookPanelProps) =
...
       ui.setNotification({
         category: 'success',
         message: `Successfully updated webhook "${res.name}"`,
       })
+      setIsSubmitting(false)
```
Comment (MildTomato) [inline studio/components/interfaces/Database/Hooks/EditHookPanel.tsx:80]: this a mistake?

File: studio/data/database-triggers/database-trigger-update-transaction-mutation.ts [hunk 4]
```diff
@@ -41,7 +41,7 @@ export async function updateDatabaseTrigger({
 }: DatabaseTriggerUpdateVariables) {
   const sql = getDatabaseTriggerUpdateSQL({ originalTrigger, updatedTrigger })
   const { result } = await executeSql({ projectRef, connectionString, sql })
-  return result
+  return updatedTrigger
```
Comment (MildTomato) [inline studio/data/database-triggers/database-trigger-update-transaction-mutation.ts:44]: this a mistake?

Comment (shishiro26) [reply studio/data/database-triggers/database-trigger-update-transaction-mutation.ts]: Hey @MildTomato  caan u check this once now

**Comment by shishiro26** [comment]
@MildTomato, could you please review my changes.

File: studio/data/database-triggers/database-trigger-update-transaction-mutation.ts (hunk 4 above)
Comment (MildTomato) [reply studio/data/database-triggers/database-trigger-update-transaction-mutation.ts]: Yep, i think this is a mistake.

File: package-lock.json
Comment (MildTomato) [inline package-lock.json:1]: This file shouldn't have any changes in it

File: apps/docs/pages/guides/auth/learn/auth-deep-dive/auth-deep-dive-jwts.mdx
Comment (MildTomato) [inline apps/docs/pages/guides/auth/learn/auth-deep-dive/auth-deep-dive-jwts.mdx:1]: we can't change the url i don't think

Comment (shishiro26) [reply apps/docs/pages/guides/auth/learn/auth-deep-dive/auth-deep-dive-jwts.mdx:1]: @MildTomato , the issue here is that the bug exists in every other subsection within the Deep Dive section, but you mentioned changing it only for "Part one: JWT's." Therefore, if I were to create another navigation menu constants in the NavigationMenu.constants.ts file, it might appear messy. So, I decided to change the file's position within the directory and moved it to the auth section. Since Next.js follows folder routing, the page now displays for the URL "docs/learn/auth-deep-dive/auth-deep-dive-jwts." If you want these changes to apply to every other subsection in the DEEP DIVE section, I can make the adjustments without altering the URL

Comment (MildTomato) [reply apps/docs/pages/guides/auth/learn/auth-deep-dive/auth-deep-dive-jwts.mdx:1]: Easy fix could be to add the following to https://github.com/supabase/supabase/blob/1795966d245ce92df4da9c2611737297f6830c60/apps/docs/components/Navigation/NavigationMenu/NavigationMenu.tsx#L30C1-L30C24

```
  {
    id: 'auth',
    path: '/learn/auth-deep-dive',
    type: 'guide',
  },
```

Comment (shishiro26) [reply apps/docs/pages/guides/auth/learn/auth-deep-dive/auth-deep-dive-jwts.mdx:1]: This is the change I'm willing to make. If I have to fix all the other subsections in the DEEP DIVE, I'll use the approach you mentioned. However, if it's only for one subsection, I'll stick with the previous approach

**Comment by shishiro26** [comment]
@MildTomato, could you please review my changes.

File: studio/data/database-triggers/database-trigger-update-transaction-mutation.ts [hunk 5]
```diff
@@ -40,8 +40,8 @@ export async function updateDatabaseTrigger({
   updatedTrigger,
 }: DatabaseTriggerUpdateVariables) {
   const sql = getDatabaseTriggerUpdateSQL({ originalTrigger, updatedTrigger })
-  await executeSql({ projectRef, connectionString, sql })
-  return updatedTrigger
+  const { result } = await executeSql({ projectRef, connectionString, sql })
```
Comment (MildTomato) [inline studio/data/database-triggers/database-trigger-update-transaction-mutation.ts:43]: can you revert this change please?

File: studio/data/database-triggers/database-trigger-update-transaction-mutation.ts [hunk 6]
```diff
@@ -40,8 +40,8 @@ export async function updateDatabaseTrigger({
...
+  return result;
```
Comment (MildTomato) [inline studio/data/database-triggers/database-trigger-update-transaction-mutation.ts:44]: can you revert these changes?

**Review by MildTomato — DISMISSED** [review]

File: studio/data/database-triggers/database-trigger-update-transaction-mutation.ts (hunk 6 above)
Comment (shishiro26) [reply studio/data/database-triggers/database-trigger-update-transaction-mutation.ts]: "Sorry for these minor issues on my part, @MildTomato

**Comment by shishiro26** [comment]
@MildTomato  could please review my changes

**Comment by shishiro26** [comment]
@MildTomato could you please review my changes

**Review by Isaiah-Hamilton — APPROVED** [review]
